pkg/cloudprovider/aws/instancetypes.go

```diff
@@ -179,7 +179,7 @@ func (p *InstanceTypeProvider) CacheUnavailable(ctx context.Context, instanceTyp
 		capacityType,
 		InsufficientCapacityErrorCacheTTL)
 	// even if the key is already in the cache, we still need to call Set to extend the cached entry's TTL
-	p.unavailableOfferings.SetDefault(UnavailableOfferingsCacheKey(capacityType, instanceType, zone), sets.Empty{})
+	p.unavailableOfferings.SetDefault(UnavailableOfferingsCacheKey(capacityType, instanceType, zone), struct{}{})
```

Comment:
I did the `sets.Empty{}` thing because it was _you_ who asked I use a variable instead of `struct{}{}` XD

Reply:
Yeah. However, I realized that we were abusing the sets package. While we do happen to depend on it here, I didn't want our cache to by the only depender on this set package if things got refactored. I looked around for someone else who aliased it, and decided to use the raw guts.